.github/workflows/main.yml

```diff
@@ -0,0 +1,23 @@
+name: Generate config
+on:
+  push:
+    branches:
+      - master
+      - add-circle-build
+
+jobs:
+  build:
+    name: Generate
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v1
+      - uses: bahmutov/npm-install@v1
+      - run: npm run build
+      - run: git status
+      # if there is a change to the circle.yml file
+      # then commit and push it. Only do this on "master" branch
+      # https://github.com/mikeal/publish-to-github-action
+      - uses: mikeal/publish-to-github-action@master
```

Comment:
I will revisit this later - this action tries to push to `master` branch, which is incorrect for our purposes. We really need to push back to the current branch to regenerate `circle.yml` file in the current branch and then run all the jobs again.